engines/dfc_provider/spec/requests/supplied_products_spec.rb

```diff
@@ -142,6 +142,35 @@
               '"ofn:spree_product_id":90000'
             )
         end
+
+        context "when supplying spree_product_uri matching the host" do
+          it "creates a variant for the existing product" do |example|
+            supplied_product[:'ofn:spree_product_uri'] = "http://www.example.com/api/dfc/enterprises/10000/supplied_products/10001"
```

Comment:
There is probably a way to configure the specs to use `test.host` as host instead of `www.example.com`. I didn't take the time to look at it.